src/app/component/quiz/quiz.component.ts

```diff
@@ -215,17 +210,18 @@ export class Quizcomponent implements OnInit, OnDestroy {
         this.points = this.points += 1;
         this.correctanswer++;
       } else if (!selectedAnswer && correctOptions) {
-        debugger;
         this.points = this.points -= 0.25;
         this.inCorrectanswer++;
       }
-      if (questionindex === this.question.length) {
+      debugger;
```

Comment:
Could you please remove debugger from here

Reply:
yes i removed it